src/lib/sql/sql_pipeline_statement.hpp

```diff
@@ -23,11 +27,11 @@ struct SQLPipelineStatementMetrics {
 };
 
 enum class SQLPipelineStatus {
-  NotExecuted,  // The pipeline or the pipeline statement has been not been executed yet.
+  NotExecuted,  // The pipeline or the pipeline statement has not been executed yet.
   Success,      // The pipeline or the pipeline statement has been executed successfully. If use_mvcc is set but no
                 //     transaction_context was supplied, the statement has been auto-committed. If a context was
                 //     supplied, that context continues to be active (i.e., is not yet committed).
-  RolledBack    // The pipeline or the pipeline statement caused a transaction conflict and has been rolled back.
+  Failure       // The pipeline or the pipeline statement caused a transaction conflict and has been rolled back.
```

Comment:
Explicitly rolling back the transaction can happen thorugh the ROLLBACK; command by the client and is considered as correct behavior. It can be done, even if there is no transaction conflict present. Otherwise we would return the following error message to the client ` "Transaction conflict, transaction was rolled back. Failed statement: " + failed_statement}`.